docs/develop/05_dApp/verifier/03_verifyCrendentials.md

```diff
@@ -0,0 +1,12 @@
+---
+id: verify
+title: Verify Credential
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import VerifyCredentials from '!!raw-loader!@site/code_examples/workshop/verifier/verifyCredentials.ts';
+
+Request should be sent from the frontend with message from extension and session ID. This code block will use <a href="https://kiltprotocol.github.io/sdk-js/modules/_kiltprotocol_messaging.html" target="_blank"> Decrypt Messages</a> protocols from KILT SDK and returns the verified credentials. This information can be displayed through the frontend.
```

Comment:
Is this about receiving a credential from an extension?

Reply:
Yes

Reply:
should be clarified. Is this the  backend code here?